nixos/modules/services/backup/burpClient.nix

```diff
@@ -0,0 +1,65 @@
+{ config, lib, networking, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.burpClient;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.burpClient = {
+
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable burp.";
+      };
+
+      configFile = mkOption {
+        default = "/etc/burp/burp.conf";
```

Comment:
I only meant to outline how a declarative solution could be obtained. If you are familiar with the technique I described, I would much prefer it to what is currently used.